packages/office-ui-fabric-react/src/components/pickers/TagPicker/TagItem.scss

```diff
@@ -13,6 +13,11 @@
   white-space: nowrap;
   cursor: default;
   user-select: none;
+  display: flex;
+  flex-wrap: nowrap;
+  max-width: 100%;
+  position: absolute;
+  right: 2px;
```

Comment:
Yeah, there are some unnecessary styles here. I'm cleaning it up a bit. The position absolute is overriden elsewhere, so is unnecessary.